iocage_lib/ioc_list.py

```diff
@@ -268,8 +269,20 @@ def list_all(self, jails):
                     short_ip4 += "(Network Issue)"
                     full_ip4 = f"DHCP - Network Issue:{e}"
                 else:
-                    full_ip4 = f"{interface}|" \
-                        f"{out.splitlines()[2].split()[1].decode()}"
+                    try:
+                        full_ip4 = f'{interface}|' \
+                            f'{out.splitlines()[2].split()[1].decode()}'
+                    except IndexError as e:
+                        short_ip4 = '(Network Issue)'
```

Comment:
Let’s use the same DHCP - Failed Parsing message without the rest here.

Reply:
Should we log it still ? I think we should perhaps in a case where we have out but fail to parse the output of the command ? Thoughts ?

Reply:
I think in this case the list is essentially a log. If we log it, that will print and clutter things up. Until we have debug logs and a verbose mode anyways.